src/hotspot/cpu/ppc/macroAssembler_ppc_sha.cpp

```diff
@@ -23,6 +23,9 @@
 // (http://www.iwar.org.uk/comsec/resources/cipher/sha256-384-512.pdf).
 
 #include "asm/macroAssembler.inline.hpp"
+#ifdef AIX
+#include "runtime/os.hpp" // malloc
+#endif
```

Comment:
Is it actually important to make the inclusion conditional?  Also, the Style Guide says conditional includes go at the end.

Reply:
No, its not. I removed it.